Find a functionally equivalent code:android.appwidget.AppWidgetHost.deleteHost:COMMENT
Method Modifier: public      
Comment:/**
 * Remove all records about this host from the AppWidget manager.
 * <ul>
 * <li>Call this when initializing your database, as it might be because of a data wipe.</li>
 * <li>Call this to have the AppWidget manager release all resources associated with your
 * host.  Any future calls about this host will cause the records to be re-allocated.</li>
 * </ul>
 */

@@ -1,6 +1,6 @@
 {
     try {
-        sService.deleteHost(mHostId, mContext.getUserId());
+        sService.deleteHost(mContext.getOpPackageName(), mHostId);
     } catch (RemoteException e) {
         throw new RuntimeException("system server dead?", e);
     }

